account_product_fiscal_classification/models/account_product_fiscal_classification_model.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2015 -Today Akretion (http://www.akretion.com)
+#   @author Renato Lima (https://twitter.com/renatonlima)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class AccountProductFiscalClassificationModel(models.AbstractModel):
```

Comment:
@rvalyi. 
- I don't see the real interest ot this class. It's just for sharing name, active, description fields ? What about to remove it ? (It make also the active help bad).
- Do you remember the objective of the field ``code`` ? Using this module since 2 years, I don't see one.
- what about removing _max_length_name ? 

well, simplification ;-)

regards